docs/conf.py

```diff
@@ -59,12 +59,7 @@
 autoapi_type = 'python'
 autoapi_dirs = ['../src/pudl', ]
 autoapi_ignore = [
-    "*/convert/datapkg_to_sqlite.py",
-    "*/convert/merge_datapkgs.py",
-    "*/load/csv.py",
-    "*/load/metadata.py",
     "*_test.py",
-    "*/package_data/*",
```

Comment:
I think we do need to keep this line actually -- we don't want AutoAPI trying to build docs from stuff under that directory.

Reply:
lol i was a little trigger happy!